src/upload/form/wasm_module.rs

```diff
@@ -20,8 +20,8 @@ impl WasmModule {
             .ok_or_else(|| anyhow!("filename should not be empty: {}", path.display()))?;
 
         Ok(Self {
-            filename,
             path,
+            filename,
```

Comment:
What's up with this change?

Reply:
Oh, I ran `cargo clippy --fix` and I think it reordered the fields for some reason. I'll revert it.